model/build_models.py

```diff
@@ -339,19 +348,27 @@ def build_model(group_size_data, TIME, contact_matrix1, contact_matrix2,
             I_14 = delta_i[i-14]
         
         dSdt = - contacts 
-        dIdt = contacts - recovery_rates * I_14       
+        dIdt = contacts - recovery_rates * I_14 
         dRdt = recovery_rates * I_14
 
         S_t = S_t + dSdt   
         I_t = I_t + dIdt
         R_t = R_t + dRdt
         
         if i <= params.prison_peak_date:
-            I_t[white_prison_i] = np.exp(i*k1)
-            I_t[black_prison_i] = np.exp(i*k2)
+            #KEEP TRACK OF RELEASES
+            prison_infection["White"].append(np.exp(i*k1)) # number of white prisoners newly infected on day i
+            prison_infection["Black"].append(np.exp(i*k2)) # number of black prisoners newly infected on day i
             S_t[white_prison_i] = Group_Size[white_prison_i] - np.exp(i*k1)
             S_t[black_prison_i] = Group_Size[black_prison_i] - np.exp(i*k2)
-          
+        elif i > params.prison_peak_date:
+            prison_infection["White"].append(prison_infection["White"][-1])
+            prison_infection["Black"].append(prison_infection["Black"][-1])
+        if i > 9: # make sure only the last ten days are kept in the infections lists
```

Comment:
this isn't the last ten days ... its every day after the first ten days? was that intentional

Reply:
I think its find but then you shouldn't sum everything in line 370, you should be just doing the last ten days?